lib/slide_hub/convert_procedure.rb

```diff
@@ -110,14 +110,16 @@ def upload_files
     def update_database
       if ENV.fetch('OSS_MULTI_TENANT') { false }
         tenants = Tenant.pluck(:name)
+        logger.info("all tenants are #{tenants.to_s}")
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Lint/StringConversionInInterpolation: Redundant use of `Object#to_s` in interpolation.